src/pages/Learn/Lesson.jsx

```diff
@@ -99,29 +100,14 @@ const Lesson = ({ lesson }) => {
             {expandedKey === key && (
               <div className={styles.descriptionContent}>
                 {key === MEDIA_TYPES.VIDEO ? (
-                  <>
-                    <video
-                      controls
-                      src={description[key]}
-                      className={styles.videoPlayer}
-                    >
-                      Your browser does not support the video tag.
-                    </video>
-                    <a
-                      href={description[key]}
-                      download
-                      className={styles.downloadLink}
-                    >
-                      Download Video
-                    </a>
-                  </>
+                  <Video videoLink={description[key]} />
                 ) : key === MEDIA_TYPES.IMAGE ? (
                   <div className={styles.imageWrapper}>
                     <img
                       src={description[key]}
                       alt={key}
                       className={styles.imageDisplay}
-                      style={{ transform: `scale(${zoomLevel})` }}
+                      style={{transform: `scale(${zoomLevel})` }}
                     />
                     <div className={styles.zoomControls}>
                       <button
```

Comment:
Yesterday, I changed the code in the Lesson.jsx file. Changing the code in LessonController.mjs and LessonRoutes.mjs is fine, but please don't modify anything in the Lesson.jsx file. It will add this line of code in Lesson.jsx file '<Video videoLink={description[key]} />' to my Lesson.jsx file.